starfish/image/_segmentation/__init__.py

```diff
@@ -47,26 +48,7 @@ def _cli(cls, args, print_help=False) -> None:
         print('Segmenting ...')
         hybridization_stack = ImageStack.from_path_or_url(args.hybridization_stack)
         nuclei_stack = ImageStack.from_path_or_url(args.nuclei_stack)
-        regions = instance.run(hybridization_stack, nuclei_stack)
-        geojson = regions_to_geojson(regions, use_hull=False)
-
-        print("Writing | regions geojson to: {}".format(args.output))
-        with open(args.output, "w") as f:
-            f.write(json.dumps(geojson))
-
-
-def regions_to_geojson(r, use_hull=True) -> List[Dict[str, Dict[str, Any]]]:
-    """Convert region geometrical data to geojson format"""
-
-    def make_dict(id_, verts) -> Dict[str, Dict[str, Any]]:
-        d = dict()
-        c = list(map(lambda x: list(x), list(map(lambda v: [int(v[0]), int(v[1])], verts))))
-        d["properties"] = {"id": id_}
-        d["geometry"] = {"type": "Polygon", "coordinates": c}
-        return d
+        label_image = instance.run(hybridization_stack, nuclei_stack)
 
-    if use_hull:
-        coordinates = r.hull
-    else:
-        coordinates = r.coordinates
-    return [make_dict(id_, verts) for id_, verts in enumerate(coordinates)]
+        print(f"Writing label image to {args.output}")
+        imsave(args.output, label_image)
```

Comment:
A bit concerned that our output format for this stage is a numpy array...

Reply:
I believe an array is the field standard. We could wrap this up in an xarray. The image should have x, y, and z, as well as corresponding coordinates.